src/io.js

```diff
@@ -6,7 +6,7 @@ import sgp4init from './propagation/sgp4init';
 
 /* -----------------------------------------------------------------------------
  *
- *                           function twoline2rv
+ *                           function twoline2satrec
```

Comment:
Can you explain why you are updating the name? Did you update the comments above too?

Reply:
Because classically, `rv` stands for satellite state vector: [rx, ry, rz, vx, vy, vz]. This represents the position and velocity in the x,y and z directions. But this function actually returns a `satrec` object. 

This is not the same as a satellite state vector. A `satrec` object is a dictionary containing all the `sgp4` satellite information [[sgp4io.cpp]](https://os.mbed.com/users/gradyhillhouse/code/SGP4_vallado/file/806c1dd6946c/sgp4io.cpp/).

I have not modified any other comments above. I figured that the old way on naming the function was not in-line with its modern interpretation of the term `rv` and `satrec`.

Reply:
I couldn't see the whole code when originally commenting - you are 100% correct.